state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1057,6 +1078,16 @@ pub mod pallet {
 						MinimumNetworkFee::<T>::set(min_fee);
 						Self::deposit_event(Event::<T>::MinimumNetworkFeeSet { min_fee });
 					},
+					PalletConfigUpdate::SetInternalSwapNetworkFee { fee } => {
+						InternalSwapNetworkFee::<T>::set(fee);
+						Self::deposit_event(Event::<T>::InternalSwapNetworkFeeSet { fee });
+					},
+					PalletConfigUpdate::SetInternalSwapMinimumNetworkFee { min_fee } => {
+						InternalSwapMinimumNetworkFee::<T>::set(min_fee);
+						Self::deposit_event(Event::<T>::InternalSwapMinimumNetworkFeeSet {
+							min_fee,
+						});
+					},
```

Comment:
Maybe it's time to consolidate these events into a single one like we did in other PRs?